openmls/src/group/core_group/test_external_init.rs

```diff
@@ -186,15 +211,30 @@ fn test_external_init(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProv
     // Now we assume that Bob somehow lost his group state and wants to add
     // themselves back through an external commit.
 
-    // Have Alice export everything that Bob needs.
-    let pgs_encoded: Vec<u8> = group_alice
-        .export_public_group_state(backend, &alice_credential_bundle)
-        .expect("Error exporting PGS")
-        .tls_serialize_detached()
-        .expect("Error serializing PGS");
-    let verifiable_public_group_state =
-        VerifiablePublicGroupState::tls_deserialize(&mut pgs_encoded.as_slice())
-            .expect("Error deserializing PGS");
+    let group_info = {
```

Comment:
Looks like you do this a couple times. Can you factor it out to avoid code duplication

Reply:
:+1: